spp_change_request/i18n/fr.po

```diff
@@ -635,12 +635,6 @@ msgstr ""
 "Erreur dans les étapes de validation. Aucune étape disponible à affecter à "
 "cette validation."
 
-#. module: spp_change_request
```

Comment:
@gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

Reply:
It should be kept @kneckinator and later be updated to match the change in data/dms.xml